tests/chainer_tests/initializer_tests/test_sampling.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+from chainer.backends import cuda
+from chainer import initializers
+from chainer import testing
+from chainer.testing import attr
+import numpy
+
+
+@testing.parameterize(*testing.product({
+    'target': [
+        initializers.UpsamplingDeconvFilter,
+        initializers.DownsamplingConvFilter,
+    ],
+    'interpolation': ['bilinear'],
+    'shape': [(5, 5, 3, 3), (5, 1, 3, 3)],
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestUpsamplingDeconvFilter(unittest.TestCase):
+
+    def check_initializer(self, w):
+        initializer = self.target(
+            interpolation=self.interpolation, dtype=self.dtype)
+        initializer(w)
+        self.assertTupleEqual(w.shape, self.shape)
+        self.assertEqual(w.dtype, self.dtype)
+
+    def test_initializer_cpu(self):
+        w = numpy.empty(self.shape, dtype=self.dtype)
+        self.check_initializer(w)
```

Comment:
This test can pass even if the initializer does nothing since shape and dtype are correctly initialized.

Reply:
I update to check initializer output sum.
If initializer doe nothing, its test fails.